### ISSUE TITLE
Prebuild binaries for Node 10 not available

### ISSUE THREAD

**Issue by interpretor** [kelektiv/node.bcrypt.js#690]
The prebuilt binaries for Node 10 (v64) are missing.
Shouldn't they be there because of the LTS support?

**Comment by agathver** [comment]
There's some strange error that is preventing the upload. I'm looking into it.

**Comment by interpretor** [comment]
The Travis-CI job now claims that the ReleaseAsset already exists, so maybe the fix would be to delete the release and rerun all jobs?

```
752 Error: {"message":"Validation Failed","request_id":"B554:57C7:14F81C:18F4CD:5C28D80D","documentation_url":"https://developer.github.com/v3","errors":[{"resource":"ReleaseAsset","code":"already_exists","field":"name"}]}
```

**Comment by brandonburkett** [comment]
I hit this today with CircleCI builds and rolled back to 3.0.2 for now.

**Comment by aluramh** [comment]
Same. I also rolled back to 3.0.2 and stopped getting errors on Windows 10 x64.

**Comment by djanowski** [comment]
3.0.2 does work on Node 10 / macOS.

**Comment by wagnerteixeira** [comment]
same error in Linux 4.15.0-29deepin-generic x86_64 GNU/Linux

**Comment by agathver** [comment]
Windows builds available on 3.0.4 onwards
